### PR TITLE
Add initial Windows support

### PR DESCRIPTION
**Pull request by TooTallNate**
* Fixes `PATH` env var delimiter for Windows (`;` vs. `:`)
* On Windows, invoke `bootstrap.js` files using Node because we can't rely on `sh` being available
* Only the `nodejs`-related runtimes are working at the moment, hence _initial_ Windows support
* Process (un)freezing is not supported on Windows

### CODE REVIEW

File: src/providers/native/index.ts [hunk 1]
```diff
@@ -58,7 +59,10 @@ export default class NativeProvider implements Provider {
 	async createProcess(): Promise<ChildProcess> {
 		const { runtime, params, region, version, extractedDir } = this.lambda;
 		const binDir = join(runtime.cacheDir, 'bin');
-		const bootstrap = join(runtime.cacheDir, 'bootstrap');
+		const bootstrap = join(
+			runtime.cacheDir,
+			isWin ? 'bootstrap.js' : 'bootstrap'
```
Comment (styfle) [inline src/providers/native/index.ts]: Why not always use `bootstrap.js`? Why is Windows different here?

**Comment by rauchg** [comment]
Is Windows CI working?

File: src/providers/native/index.ts (hunk 1 above)
Comment (rauchg) [reply src/providers/native/index.ts]: This is so bizarre that it would definitely warrant a comment

Comment (TooTallNate) [reply src/providers/native/index.ts]: Basically there's two implementations of these "bootstrap" logic. Ideally just temporarily, and we use the .js ones for all platforms, but for now running the .js ones on unix was causing tests to fail. So consider this just an in-progress transition to that while we get Windows initially/incrementally working.

**Comment by TooTallNate** [comment]
@rauchg Not yet, but #13 is a start at it. Not all the tests are passing on Windows yet anyways, but I want to get this merged so that we can get `@now/node` working on `now dev` at least.

**Pull request merged**